src/components/modal/modal.tsx

```diff
@@ -190,6 +190,7 @@ export class Modal
     connectConditionalSlotComponent(this);
     connectLocalized(this);
     connectMessages(this);
+    connectFocusTrap(this);
```

Comment:
Does `connectFocusTrap` need to be updated to clean up existing focus traps?

Reply:
I don't think so. There's no destroy method in the focus-trap. Its already getting deactivated on disconnectedCallback.